tests/integration/controller/RegisterControllerTest.php

```diff
@@ -78,7 +79,7 @@ public function setUp () {
 		$this->groupManager = \OC::$server->getGroupManager();
 		$this->defaults = $this->createMock(Defaults::class);
 		$this->random = \OC::$server->getSecureRandom();
-		$this->usersession = $this->createMock(IUserSession::class);
+		$this->usersession = $this->createMock(UserSession::class);
```

Comment:
I had to change this to `OC\User\Session` so the mock object implements the ` createSessionToken` method (which the `IUserSession` doesn't define).